streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -238,31 +221,30 @@ private void validateStoreOpen() {
         }
     }
 
-    private byte[] getInternal(byte[] rawKey) {
+    private byte[] getInternal(final byte[] rawKey) {
         try {
             return this.db.get(rawKey);
-        } catch (RocksDBException e) {
-            throw new ProcessorStateException("Error while getting value for key " + serdes.keyFrom(rawKey) +
-                    " from store " + this.name, e);
+        } catch (final RocksDBException e) {
+            throw new ProcessorStateException("Error while getting value for key from store " + this.name, e);
```

Comment:
This change is intentional. Same below removing raw key/value from the exception error message. Cf. https://issues.apache.org/jira/browse/KAFKA-6538 as follow up for this change.